packages/ddp-client/livedata_connection_tests.js

```diff
@@ -2,7 +2,10 @@ var newConnection = function (stream) {
   // Some of these tests leave outstanding methods with no result yet
   // returned. This should not block us from re-running tests when sources
   // change.
-  return new LivedataTest.Connection(stream, {reloadWithOutstanding: true});
+  return new LivedataTest.Connection(stream, {
+    reloadWithOutstanding: true,
+    bufferedWritesInterval: 0
```

Comment:
I've disabled buffering for the tests, as they all written to expect synchronous behaviour.

Not sure if we're better adding new tests or trying to adapt/modify existing ones.